### DISCUSSION TITLE
Release 1.2.0 Beta 15

### DISCUSSION THREAD

**Discussion by apohl79** [apohl79/audiogridder#1059] [General]
## PLUGIN
- **NEW** Added support for custom block sizes: This allows using larger block sizes than the host and might help with high latency chains.
- Fixed a buffering issue that caused damage to the audio signal
- Fixed automatic retries for chain sandbox load timeouts
- Fixed reconnect loop issue when "bypass when not ready" was disabled
- Fixed PDC issue when bypassing plugins
- Stability fixes

## SERVER
- Stability fixes

All downloads are available at https://audiogridder.com/download/.

<hr /><em>This discussion was created from the release <a href='https://github.com/apohl79/audiogridder/releases/tag/release_1_2_0_beta15'>Release 1.2.0 Beta 15</a>.</em>

**Comment by tbffbt** [comment]
I've come to understand that, at least for me, the infamous above-1000% CPU peaking issue is related to beta versions. Perhaps because I run what you could call a sub-optimal setup with an outdated MacOS Mojave 10.14.6. I try every beta version as soon as they're released but none of them is usable, so I need to stick to stable versions.
Is there a specific reason for that? Apart, of course, from the development state. Sadly I cannot update my OS out of compatibility, but I guess my question is: should I expect full compatibility with my setup for the next stable version?

**Reply by apohl79** [reply]
I suggest you start a new help discussion on this topic and provide as much details as possible. (screen recordings or screenshots showing the problem, also see https://audiogridder.com/bug-reports/). 

I don't have any open issues on the list right now for the next stable version. Just monitoring crashes and checking bug reports. I will rely on details from you to find the cause of your issue, as there is no way for me to replicate your setup.

**Reply by giacometti777** [reply]
See my comment below about the possibility WiFi is derailing your performance! Just a thought!

**Comment by fojerhar** [comment]
HI Andreas

PDC bypass issue gone, beautiful!!!!
THANK YOU ( I don't have bigger letters for typing this out thank capitols)

**Comment by tobumusikizm** [comment]
I also have cpu spikes.I run it on the crossgridder. I have just downloaded beta 15.But the plug in still shows beta 14.Thank you for the great application.
peace

**Comment by MM502** [comment]
Running smoothly here on W10|Reaper 6.86 with Server on self and i10780H laptop. Thank you so much for your efforts - I have sent a donation.

**Comment by giacometti777** [comment]
A bug that seems possible to fix that I've encountered:

I have a PC connected as a server via usb-c to ethernet adapter to a Mac running AG plugin. When reopening projects - I believe AG sometimes utilizes the WiFi connection rather than the manually specified IP based on the server identity. So the AG plugin loads "PC" as a server but there are huge CPU spikes until I manually specify the IP again.

**Comment by peterpander** [comment]
will try this beta hope it works! 

I just want to say thank you so much @apohl79 , you've made an amazingly simple functional piece of software that has allowed me to use plugins I would otherwise have not been able to! I actually turned off image capture and use my slave as the editor of the VST and it works great with input director (using one mouse with two computers)

I will be donating if this continues to work over coming months and I would 100% pay for a full stable version - VEPRO7 is really expensive and does not even support VST3 yet..

**Comment by jingxingwenhua** [comment]
Hello, the preset file in the plug-in preset menu does not respond after clicking, and the preset cannot be selected in the preset menu. The beta6 version does not have this problem, but this problem exists in beta11-beta15. Thanks for developing such a great tool.

**Comment by fojerhar** [comment]
Hi Andreas

Can this fixed outbound buffers options made to be off by default?
Maybe it is written somewhere, but I have not found it. My system works better without It

thank you

**Reply by apohl79** [reply]
If you turn it off once, it stays disabled. no?

**Reply by fojerhar** [reply]
Yes, of course it stays off as it is turned off. It just would be a good time saver as opening the new instances.

**Reply by apohl79** [reply]
Yeah what I mean is, you turn it off, and it will be turned off for all future instances as well. Do you have individual buffer settings per plugin enabled?

**Reply by fojerhar** [reply]
Yes, individual buffer sizes are enabled, that is really useful for playing with delay compensation.

**Reply by apohl79** [reply]
Ok. With this enabled, the plugin does not store anything to the config file. 

Disable the by plugin buffer settings, now disable the fixed buffer (the config file will be updated now) and enable the individual buffer settings again.

**Reply by fojerhar** [reply]
Ahh, cool, got it, thank you (again)
